addons/test_mail/tests/test_performance.py

```diff
@@ -832,6 +832,38 @@ def test_message_format(self):
             for message in res:
                 self.assertEqual(len(message['attachment_ids']), 2)
 
+    @mute_logger('odoo.tests', 'odoo.addons.mail.models.mail_mail', 'odoo.models.unlink')
+    @users('emp')
+    @warmup
+    def test_message_format_query(self):
```

Comment:
What we usually do with query count changes is to make a preliminary commit to introduce (or update an existing) query count test, and then we make the commit with the changes and we just adapt the query count accordingly. This way it is easy to see the actual impact on number of queries of the change.

Reply:
Should be good now !